src/lib.rs

```diff
@@ -133,6 +137,8 @@ pub struct Builder {
     pub headers: HashMap<String, String>,
     /// Max retries
     pub max_retries: usize,
+    /// Async runtime, trait must implement `sleep` function, default is `tokio`
+    marker: PhantomData<S>,
 }
```

Comment:
@ValuedMammal moving Sleeper to only the `async` feature makes this not possible